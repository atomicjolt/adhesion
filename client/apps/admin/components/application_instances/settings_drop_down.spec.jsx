import React from 'react';
import { shallow } from 'enzyme';
import SettingsDropDown from './settings_drop_down';

describe('application instance modal', () => {

  let result;

  beforeEach(() => {
    result = shallow(<SettingsDropDown />);
  });

  it('renders a list with settings', () => {
    expect(result).toMatchSnapshot();
  });

});
