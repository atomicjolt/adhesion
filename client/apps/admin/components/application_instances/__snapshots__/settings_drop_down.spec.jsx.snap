// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`application instance modal renders a list with settings 1`] = `
<ul
  style={
    Object {
      "backgroundColor": "white",
      "border": "1px solid black",
      "borderRadius": "5px",
      "display": "inline-block",
      "fontSize": "2em",
      "padding": "5px",
      "position": "absolute",
    }
  }
>
  <li>
    Settings
  </li>
  <li>
    Other Settings
  </li>
  <li>
    More Settings
  </li>
</ul>
`;
